test/controllers/beneficial.owner.gov.controller.spec.ts

```diff
@@ -104,27 +104,31 @@ describe("BENEFICIAL OWNER GOV controller", () => {
 
     test(`redirects to the ${config.BENEFICIAL_OWNER_TYPE_URL} page`, async () => {
       mockPrepareData.mockReturnValueOnce(BENEFICIAL_OWNER_GOV_OBJECT_MOCK);
```

Comment:
@markpit you mentioned doing this causes an issue but seems to be ok in this test. We should investigate why.

Reply:
It's because we are using the same mock data to send into the request as we are to return from mock functions. When we return the from the mock, some functions will then modify the data. In my case it was because the test was to check if service address got removed. The mock data had the radio button set appropriately but the after the test completes, the data had been changed during the test, meaning that when that same mock was used again, it failed validation.

Reply:
in your case they are different mocks, but in the one I was fixing it was the same one